src/main/kotlin/no/nav/syfo/infrastructure/database/repository/VurderingRepository.kt

```diff
@@ -74,9 +88,25 @@ class VurderingRepository(private val database: DatabaseInterface) : IVurderingR
             it.executeQuery().toList { toPVarselPdf() }.single()
         }
 
+    private fun Connection.getVarselForVurdering(vurdering: PVurdering): Varsel? =
+        prepareStatement(GET_VARSEL_FOR_VURDERING).use {
+            it.setInt(1, vurdering.id)
+            it.executeQuery().toList { toPVarsel() }
+        }.map { it.toVarsel() }.firstOrNull()
+
     companion object {
         private val mapper = configuredJacksonMapper()
 
+        private const val GET_VURDERING =
+            """
+                SELECT * FROM VURDERING WHERE personident=? ORDER BY created_at DESC
```

Comment:
Må vel også ha `AND TYPE = FORHANDSVARSEL`, evt må man filtere ut dette i kotlin-koden?

Reply:
Evnt ikke hvis vi generaliserer api'et?